addons/point_of_sale/__manifest__.py

```diff
@@ -105,82 +91,56 @@
             ("include", "web._assets_bootstrap"),
             ("include", "web._assets_bootstrap_backend"),
             ('include', 'web._assets_core'),
-            ("remove", "web/static/src/core/browser/router.js"),
```

Comment:
I remembered that it's important to remove this router.js in pos to make the backend views work properly. Is it not included after this refactoring?